README.md

```diff
@@ -19,28 +19,32 @@ This crate is `#[no_std]` compatible with `default-features = false`
 | Feature                | Default? | Description |
 | :---                   | :---     | :---        |
 | `alloc`                | ✓        | Enables features that require dynamic heap allocation |
-| `std`                  | ✓        | std::error::Error types |
-| `zeroize`              | ✓        | Enables `Zeroize` for `SigningKey` |
-| `asm`                  |          | Assembly implementation of SHA-2 compression functions |
-| `batch`                |          | Batch verification. Requires `alloc` |
-| `digest`               |          | TODO |
-| `legacy_compatibility` |          | See: A Note on Signature Malleability |
-| `pkcs8`                |          | PKCS#8 Support |
-| `pem`                  |          | PEM Support |
-| `rand_core`            |          | TODO        |
+| `std`                  | ✓        | Implements `std::error::Error` for `SignatureError` |
+| `zeroize`              | ✓        | Implements `Zeroize` and `ZeroizeOnDrop` for `SigningKey` |
+| `rand_core`            |          | Enables `SigningKey::generate` |
+| `batch`                |          | Enables `verify_batch` for verifying many signatures quickly. Also enables `rand_core`. |
+| `digest`               |          | Enables `Context`, `SigningKey::{with_context, sign_prehashed}` and `VerifyingKey::{with_context, verify_prehashed, verify_prehashed_strict}` for Ed25519ph prehashed signatures |
+| `asm`                  |          | Enables assembly optimizations in the SHA-512 compression functions |
```

Comment:
I re-ordered this alphabetically between default/optionals - just wonderign what is the ordering here so they are easily found when looked up ? also legacy_compatibility last ?

Reply:
I think alphabetical is fine. The previous ordering was thematic, where the themes are things only I had in my head :)